src/package.json

```diff
@@ -55,7 +55,8 @@
 			"@mikro-orm/knex": "6.4.16",
 			"@mikro-orm/postgresql": "6.4.16",
 			"@mikro-orm/mysql": "6.4.16",
-			"@mikro-orm/sqlite": "6.4.16"
+			"@mikro-orm/sqlite": "6.4.16",
+			"graphql": "16.11.0"
```

Comment:
This shouldn't have any real effect on graphweaver itself, as they already reference this version,  but it's necessary to make pulling in an app like Curby reliable.